src/coord/src/error.rs

```diff
@@ -324,6 +329,13 @@ impl fmt::Display for CoordError {
                 value.quoted(),
                 reason,
             ),
+            CoordError::InvalidReplicaSize { size, expected } => {
+                write!(
+                    f,
+                    "unknown cluster size {size}. Expected one of: {}.",
```

Comment:
We try to follow the [PostgreSQL error style guide](https://www.postgresql.org/docs/current/error-style-guide.html) now. The "expected one of" should be in the hint field (see `fn hint` below).

The usual PostgreSQL style for this sort of hint, btw, is: "Valid cluster sizes are:  size1, size2, size3"